<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Launcher</class>
 <widget class="QMainWindow" name="Launcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Launcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnAction">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="prDownload">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Status Label</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Title Label</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>397</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
